Tests/ChessGameTest.php

```diff
@@ -950,6 +950,40 @@ public function testGameShouldNotBeOverInClaimableDraw()
         $this->assertRepetitionDraw();
     }
 
+    public function testGameShouldEndInFiveFoldRepetition()
```

Comment:
i think a good test here would be to include a test where a pawn has just moved 2 squares and therefore an en passant position may exist. Or one where castling has or hasnt happened, but king and rook are in same position. Reason I say that is because if you're just comparing fen strings to ensure they are equal, seems like there might be times where the fen is different but the position is the same.